src/components/media-loader.js

```diff
@@ -67,6 +67,8 @@ AFRAME.registerComponent("media-loader", {
 
       const { raw, origin, contentType } = await resolveMedia(url, token);
 
+      this.el.emit("media_resolved", { src: this.data.src, raw, origin, contentType });
+
```

Comment:
With PDFs this block of code is going to be hit for every page. Are we going to want to filter that?